RHEL6_7/databases/postgresql/check

```diff
@@ -236,7 +236,7 @@ EOF
   See the upstream article for pg_dumpall:
   [link:$UPSTREAM_DOC_PGDUMP]
 EOF
-    log_slight_risk "We cannot tell for 100% that the system will be in-place upgradeable."
+    log_info "We cannot tell for 100% that the system will be in-place upgradeable."
     DIFFERENT_USAGE_OK=yes
     return $RESULT_PASS
```

Comment:
It seems that `$RESULT_PASS` smells here. Read the two messages. I am thinking even whether we can ends with `$RESULT_PASS` ever, because it seems more like that this test is about `$RESULT_FAIL` and `$RESULT_INFORMATIONAL` as in case of pass, you will not see the solution text even. But in that case it is bug that it is already part of upstream for a long time.

Reply:
I think this is a question of a problem scope. Do we define `$RESULT_PASS` as a total success of the upgrade/migration process, without any chance of fail? Or do we define it as a success in the scope, what we are able to provide with preupgrade assistant? If the latter is a case, then I think the handling of the actual user data content of the database is outside of this scope and `$RESULT_PASS` is justified.

Reply:
Agree. It would be better to use just something like 0 / 1 instead of `$RESULT_*` variables in that case. But it doesn't matter in this PR.